uisdk/build.gradle

```diff
@@ -29,16 +29,16 @@ android {
             file("../app/secure.properties")?.withInputStream { secureProps.load(it) }
         }
         release {
-            buildConfigField("String", "KARHOO_WEB_TRACKING_URL", '"https://agent-portal.karhoo.com/follow/follow/"')
+            buildConfigField("String", "KARHOO_WEB_TRACKING_URL", '"https://follow.karhoo.com/"')
```

Comment:
Can you please confirm this link with Web Traveller team?

Reply:
Checked with Web. They say the links are fine